vllm_ascend/quantization/w8a8_dynamic.py

```diff
@@ -703,11 +707,17 @@ def apply(
 
         topk_weights = topk_weights.to(x.dtype)
 
+        w1_weight = layer.w13_weight
+        #We only enable weight prefetch in torch_air mode, and benchmark shows no performance gain in enforce_eager mode.
+        if self.torchair_graph_enabled:
+            torch_npu.npu_prefetch(w1_weight, topk_weights,
```

Comment:
Can you provide a profile screenshot to prove that this `CMO` instruction is completely masked by the `DynamicQuant` operator under all `FusedMOEStates`?